src/plugins/borrow-plugins/plugins/aave/util/addressToTokenId.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export const addressToTokenId = (address: string) => address.slice(2).toLowerCase()
```

Comment:
This is a problem. The only way to look up a token should be by visiting the core. Yes, we happen to have secret knowledge about how the Ethereum plugin chooses its tokenIds's, but this sort of thing should *never* be happening in the GUI.

We should probably add an `EdgeAccount.currencyConfig[pluginId].getTokenId(token: EdgeTokenInfo)`.

However, I think the real solution is to scan the wallet for available tokens, and just use the tokenId's from those. If the contract returns a token that doesn't exist in the wallet, we can't show it anyhow, and the app is likely to crash if we try.